extension/src/test/suite/experiments/index.test.ts

```diff
@@ -736,6 +737,7 @@ suite('Experiments Test Suite', () => {
     }).timeout(WEBVIEW_TEST_TIMEOUT)
 
     it("should be able to handle a message to modify an experiment's params and run a new experiment", async () => {
+      stub(DvcReader.prototype, 'listStages').resolves('train')
```

Comment:
[I] much better to get the `dvcReader` back from `buildExperiments` and stub the instance instead of the prototype. Can even move the stub back into `buildExperiments` if it is going to be required everywhere.

Reply:
Unfortunately, this does not work. I'm not sure if it's because it does not go through that same `dvcReader` or something else. I can investigate and change as a follow up.